demos/tutorial/indexed_fasta_io/solution1.cpp

```diff
@@ -25,7 +25,7 @@ int main(int argc, char const ** argv)
     if (!save(faiIndex, toCString(faiFilename)))
     {
         std::cerr << "ERROR: Could not write the index to file!\n";
-        return 1;
+        return 0;
     }
 
     std::cout << "Index file " << faiFilename << " was successfully created.\n";
```

Comment:
when I build and run the files, the stdouts are empty.
Lets see if the added .stderr files solve the problem.